Makefile

```diff
@@ -156,10 +156,22 @@ endif
 endif
 
 # Enable libfido2 for buildbox and local environments that have it.
-# TODO(codingllama): Build static binaries with libfido2.
 ifneq ("$(shell ls {/opt/homebrew,/usr/lib/x86_64-linux-gnu,/usr/local/lib}/libfido2.* 2>/dev/null)","")
 LIBFIDO2_TAG := libfido2
+else
 endif
+# Build tsh with static libfido2 support if the requirements are present in the
+# system. The required libs are uncommon enough that their presence makes for a
+# strong signal.
+LIBFIDO2_MESSAGE := without libfido2
+ifneq ("$(wildcard /usr/local/lib/libfido2.a)", "")
+ifneq ("$(wildcard /usr/local/lib/libcbor.a)", "")
+ifneq ("$(wildcard /usr/local/lib/libudev.a)", "")
```

Comment:
Maybe out of scope for this PR but it's a bit unfortunate we hard-code `/usr/local/lib` which isn't very sensible on multiarch systems. I'm not sure I have a better solution to offer though since libcbor doesn't seem to play with pkg-config or anything.

Reply:
I'm not in love this, but at least it gets us out of the door. There's definitely room for improvement.

Reply:
FWIW I get a pkg-config entry for libcbor and libfido2 when installing them on macOS through brew

Reply:
👍 

I was taking a look at pkgconfig over the weekend, I'll see if I carve some time to give it a try, specially for the static builds.